### PR TITLE
Fix build issues in VS 2013

### PR DESCRIPTION
**Pull request by lotem**
Would close Issue 108 and Issue 109

### CODE REVIEW

**Comment by lotem** [comment]
The Travis CI build may have stopped before actually building the project.
The build log ends with `sudo npm install -g node-gyp`
@BYVoid could you please have a look?

**Comment by jakwings** [comment]
Add 'sudo: required' to `.travis.yml` and try again. Ref: http://docs.travis-ci.com/user/workers/container-based-infrastructure/

**Comment by lotem** [comment]
Will this work? Trying with 'sudo: required'.

File: src/PhraseExtract.cpp [hunk 1]
```diff
@@ -1,4 +1,4 @@
-/*
+﻿/*
```
Comment (lotem) [inline src/PhraseExtract.cpp]: Added UTF-8 BOM.

**Comment by lotem** [comment]
I think this CL is ready.
PTAL @BYVoid

**Comment by BYVoid** [comment]
Thank you!

**Pull request merged**